### PR TITLE
Feat:videolist page

### PR DESCRIPTION
**Pull request by shubhamkumar648**
### file to check

- src/page/videoListing/videoListing.jsx
- src/context/Videocontext.js
chrome-capture-2022-5-2 (1)

- src/reducer/ReducerFun.js
- src/utils/filter.js

### CODE REVIEW

File: src/reducer/ReducerFun.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+export const ReducerFun = (state, action) => {
+  console.log(action.payload);
```
Comment (RahulSingh9131) [inline src/reducer/ReducerFun.js]: remove console here

File: src/routes/Route.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import React from "react";
+import { Routes,Route } from "react-router-dom";
+import { HomePage, VideoListing } from "../page";
+
+export const RoutePath = () => {
+  return (
+    <Routes>
+      <Route path="/" element={<HomePage />} />
+
+      <Route path="videolist" element={<VideoListing />} />
```
Comment (RahulSingh9131) [inline src/routes/Route.js]: i think it should be /videolist in the path.

**Review by RahulSingh9131 — APPROVED** [review]
* context for videos is fine
* css is looking good too
if possible give some padding around context-description in videocard.

File: src/routes/Route.js (hunk 2 above)
Comment (shubhamkumar648) [reply src/routes/Route.js]: ok will do

File: src/reducer/ReducerFun.js (hunk 1 above)
Comment (shubhamkumar648) [reply src/reducer/ReducerFun.js]: ok thanks

File: src/component/header/Header.js [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+export const Header = () => {
+  return (
+    <div className="Home">
+
+    <div className="left_container ">
+    
+       <Link to="/"  className="link__nostyle ">
+       <h3 className="brandName"> 
+       namaste~Stream
```
Comment (vaibhav-mahalle) [inline src/component/header/Header.js:13]: formatting could be better you can use prettier for formatting.

**Review by vaibhav-mahalle — APPROVED** [review]
remove console.log from the code and format it using prettier. Everything else looks good.

**Pull request merged**